jaeger-core/src/main/java/com/uber/jaeger/senders/UDPSender.java

```diff
@@ -85,14 +92,20 @@ int getSizeOfSerializedSpan(Span span) throws SenderException {
    */
   @Override
   public int append(Span span) throws SenderException {
-    int spanSize = getSizeOfSerializedSpan(span);
+    if (process == null) {
```

Comment:
nit: as far as I can tell, it's not used until the call to emitBatch, so I would (a) move this to a function and (b) inline it